hera_mc/geo_handling.py

```diff
@@ -144,38 +144,44 @@ def is_in_geo_location(self, station_name):
             station_present = False
         return station_present
 
-    def is_in_connections(self, station_name, active_date=None):
+    def is_in_connections(self, station_name, active_date=None, return_antrev=True):
         """
         checks to see if the station_name is in the connections database (which means it is also in parts)
+        if active_date is None, it will return True/False if ever connected
+        if active_date is given, it will check whether it was connected at that time
 
-        return True/False unless active_date is a Time, when it returns the antenna number at that location
+        return True/False unless (1) active_date is provided and (2) return_antrev is True
 
         Parameters:
         ------------
-        args:  needed arguments to open database and set date/time
         station_name:  string name of station
-        active_date:  astropy Time to check if active, ignores if not an instance of Time
+        active_date:  astropy Time to check if active, default is None
+        return_antrev:  boolean lag to return True/False or antrev tuple, default is True
         """
 
         active_date = cm_utils._get_datetime(active_date)
-        connected_station = self.session.query(part_connect.Connections).filter(part_connect.Connections.upstream_part == station_name)
-        if connected_station.count() > 0:
-            station_connected = True
+        connected_antenna = self.session.query(part_connect.Connections).filter(part_connect.Connections.upstream_part == station_name)
+        if connected_antenna.count() > 0:
+            antenna_connected = True
```

Comment:
It might be a bit clearer if `antenna_connected` is renamed to `retval` or `result` or something. First, the name is very similar to `connected_antenna`. Second, the name is misleading when the `return_antrev` option is active.